src/services/users.service.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+const { sequelize } = require('../db');
+const { uuidv4 } = require('uuid');
+const { DataTypes } = require('sequelize');
+
+const User = sequelize.define('User', {
+  name: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
+},
```

Comment:
it also should have id, doesn't it?

Reply:
my app make table from ORM model using sync force so according to video lessons id will be made automatically, is it correct?

Reply:
yes, it is, I just thought that you've created table at first and then created model